src/internt-vedtak/FaktaOgVurderingerAktivitet.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {
+    AktivitetBarnetilsynFaktaOgVurderingerDto,
+    AktivitetLæremidlerFaktaOgVurderingerDto,
+    Vilkårperiode,
+} from './typer/vilkårperiode';
+import { Vurdering } from './Vurdering';
+import { notNullOrUndefined } from '../felles/nullOrUndefined';
+import { Stønadstype } from '../felles/stønadstype';
+
+export const FaktaOgVurderingerAktivitet: React.FC<{
+    aktivitet: Vilkårperiode;
+    stønadstype: Stønadstype;
+}> = ({ aktivitet, stønadstype }) => {
+    switch (stønadstype) {
+        case Stønadstype.BARNETILSYN: {
+            const { aktivitetsdager, lønnet } =
+                aktivitet.faktaOgVurderinger as AktivitetBarnetilsynFaktaOgVurderingerDto;
+            return (
+                <>
+                    {notNullOrUndefined(aktivitetsdager) && (
+                        <div>Aktivitetsdager: {aktivitetsdager}</div>
+                    )}
+                    <Vurdering navn={'Mottar bruker ordinær lønn i tiltaket?'} vurdering={lønnet} />
+                </>
+            );
+        }
+        case Stønadstype.LÆREMIDLER: {
+            const { prosent, studienivå, harUtgifter, harRettTilUtstyrsstipend } =
+                aktivitet.faktaOgVurderinger as AktivitetLæremidlerFaktaOgVurderingerDto;
+            return (
+                <>
+                    {notNullOrUndefined(prosent) && <div>Prosent: {prosent}</div>}
```

Comment:
Burde man vise prosent? 
```suggestion
                    {notNullOrUndefined(prosent) && <div>Prosent: {prosent} %</div>}
```